docs(stamps): clarify API key curves and fix typos in stamps page (#682)
tkhq/docs: api-reference/overview/stamps.mdx

   <Step>
      Create a JSON-encoded stamp:

>          * `publicKey`: the public key of the API key. Turnkey supports multiple API key curves:  `[API_KEY_CURVE_P256, API_KEY_CURVE_SECP256K1, API_KEY_CURVE_ED25519]`

Author: ```suggestion
         * `publicKey`: the public key of the API key. Turnkey supports multiple API key curves: `[API_KEY_CURVE_P256, API_KEY_CURVE_SECP256K1, API_KEY_CURVE_ED25519]`
```

>          * `signature`: the signature produced by the API key over the JSON-encoded POST body, hex encoded.

Author: ```suggestion
         * `signature`: the signature produced by the API key
```

>          * `publicKey`: the public key of the API key. Turnkey supports multiple API key curves, including `API_KEY_CURVE_P256`, `API_KEY_CURVE_SECP256K1`, and `API_KEY_CURVE_ED25519`. The `publicKey` value is the compressed (for P-256 and SECP256K1) or raw (for Ed25519) public key, hex encoded.

Author: ```suggestion
         * `publicKey`: the public key of the API key. Turnkey supports multiple API key curves:  `[API_KEY_CURVE_P256, API_KEY_CURVE_SECP256K1, API_KEY_CURVE_ED25519]`
```

>          * `scheme`: the signature scheme used to sign the request. The value tells the server which curve and signature algorithm to use for verification, and must match the curve of the `publicKey`. The supported schemes are:

Author: ```suggestion
         * `scheme`: the signature scheme used to sign the request, matching the curve of the `publicKey`. The supported schemes are: `[SIGNATURE_SCHEME_TK_API_P256, SIGNATURE_SCHEME_TK_API_SECP256K1, SIGNATURE_SCHEME_TK_API_ED25519]`
```

>             * `SIGNATURE_SCHEME_TK_API_P256` — for `API_KEY_CURVE_P256` keys (ECDSA over P-256, SHA-256, DER-encoded signature)

Author: ```suggestion
```

>             * `SIGNATURE_SCHEME_TK_API_SECP256K1` — for `API_KEY_CURVE_SECP256K1` keys (ECDSA over secp256k1, SHA-256, DER-encoded signature)

Author: ```suggestion
```

>             * `SIGNATURE_SCHEME_TK_API_ED25519` — for `API_KEY_CURVE_ED25519` keys (Ed25519)

Author: ```suggestion
```

   <Step>
      Create a JSON-encoded stamp:

         * `publicKey`: the public key of the API key. Turnkey supports multiple API key curves: `[API_KEY_CURVE_P256, API_KEY_CURVE_SECP256K1, API_KEY_CURVE_ED25519]`
         * `signature`: the signature produced by the API key

>          * `scheme`: the signature scheme used to sign the request, matching the curve of the `publicKey`. The supported schemes are: `[SIGNATURE_SCHEME_TK_API_P256, SIGNATURE_SCHEME_TK_API_SECP256K1, SIGNATURE_SCHEME_TK_API_ED25519]`

Author: because these are in private protos. disregarding
